sdk/keyvault/keyvault-admin/ThirdPartyNotice.txt

```diff
@@ -0,0 +1,135 @@
+Third Party Notices for keyvault-secrets
```

Comment:
Did we generate this or copy it over? Am not sure what the process has been to create this file in the past.

Reply:
Unresolving this conversation thread as I don't see a resolution here... :)

Reply:
I thought you were talking about the use of `secrets`. I did change `secrets` here. This file is copied. I think it's safe to assume that this list is about the same as of the other clients for now. We can do a deeper cleanup later, I believe.